TACC/Core-Portal
# Task/fp 318 job history modal  expand tweaks

# Notice: #

Keeping as Draft. See https://github.com/TACC/Frontera-Portal/pull/99#pullrequestreview-462385586.

- `task/FP-318-job-history-modal` #99
    - `task/FP-318-job-history-modal--ui-tweaks-more-date-expand-tweaks` #143
        - `task/FP-318-job-history-modal--ui-tweaks` #140
        - `task/FP-318-job-history-modal--expand-tweaks` #141
        - `task/FP-318-job-history-modal--more-data` #142
        - `task/FP-318-job-history-modal--re-style` #144

## Overview: ##

_(via `task/FP-318-job-history-modal`; see PR #99)_

Simplify Expand component and make styles match design.

## PR Status: ##


## Related Jira tickets: ##

* [FP-318](https://jira.tacc.utexas.edu/browse/FP-318)

## Summary of Changes: ##

_(each change is a single commit)_

- Drop utility classes. Use `Icon` component.
- Reduce markup.
- Add comments for future use of  details/summary tags.
- Style to match design.
- Add support for `className` prop. 

## Testing Steps: ##
1. Open Job History Detail View modal.
2. Ensure Expand panel still works.
2. Ensure Expand panel matches design.

## UI Photos:

[Design | Core Portal > View Details of Job](https://xd.adobe.com/view/db2660cc-1011-4f26-5d31-019ce87c1fe8-ad17/screen/7d3e671f-bb61-4a20-89c6-9ca9a78a6f93/specs/)
<img width="806" alt="FP-318 Expand Panel Style" src="https://user-images.githubusercontent.com/62723358/89512142-694ee380-d798-11ea-93ad-276d796a8b6f.png">

## Notes: ##

The [Core Portal > View Tickets](https://xd.adobe.com/view/db2660cc-1011-4f26-5d31-019ce87c1fe8-ad17/screen/e8d2c94a-feda-49b6-afac-c3967857d14d/specs/) is **not** updated. Not enough work has been done yet to support its usage of this component.
